src/components/PostsList.tsx

```diff
@@ -1,85 +1,65 @@
-import React from 'react';
+import classNames from 'classnames';
+import { Post } from '../types/Post';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+type Props = {
+  posts: Post[],
+  selectedPost: Post | null,
+  onSetSelectedPost: (post: Post | null) => void,
+};
 
-    <table className="table is-fullwidth is-striped is-hoverable is-narrow">
-      <thead>
-        <tr className="has-background-link-light">
-          <th>#</th>
-          <th>Title</th>
-          <th> </th>
-        </tr>
-      </thead>
+export const PostsList: React.FC<Props> = ({
+  posts,
+  selectedPost,
+  onSetSelectedPost,
+}) => {
+  const handlerOpenPost = (newSelectedPost: Post) => {
+    return (selectedPost === newSelectedPost)
```

Comment:
The ternary operator usage in this context is not very clear. It can be refactored for better readability by using if else statement